src/skai/model/train_lib.py

```diff
@@ -932,10 +938,11 @@ def train_and_evaluate(
     save_model_checkpoints: bool,
     save_best_model: bool,
     early_stopping: bool,
+    strategy: tf.distribute.Strategy,
```

Comment:
Please add this to the Args section in the function docstring.

Reply:
Alright, I have added the "strategy" to the docstrings. Thank you